src/core/PDFContext.ts

```diff
@@ -163,6 +163,16 @@ class PDFContext {
     throw new UnexpectedObjectTypeError(types, result);
   }
 
+  getObjectRef(pdfObject: PDFObject): PDFRef {
+    for (const [ref, object] of this.indirectObjects.entries()) {
+      if (object === pdfObject) {
+        return ref;
+      }
+    }
```

Comment:
I try to avoid changing the compiler config unless it's strictly necessary. Doing so can cause unexpected issues for library users. If we use `Array.from` we can implement this logic without changing the config.
```suggestion
    const entries = Array.from(this.indirectObjects.entries());
    for (let idx = 0, len = entries.length; idx < len; idx++) {
      const [ref, object] = entries[idx];
      if (object === pdfObject) {
        return ref;
      }
    }
```